frontend/src/components/Tabs/PatientSymptoms/ContactDoctor.js

```diff
@@ -0,0 +1,124 @@
+import React, {useEffect, useState} from 'react';
+import {useAuth} from "../../Authentication/FirebaseAuth/FirebaseAuth";
+import axios from 'axios';
+import {Button, Spinner} from 'react-bootstrap';
+import {io} from "socket.io-client";
+import {Form} from "react-bootstrap";
+
+import styles from "./ContactDoctor.module.css";
+
+function ContactDoctor(props) {
+
+    let {currentUser} = useAuth();
+    let [chats, setChats] = useState([]);
+    let [messageInput, setMessageInput] = useState('');
+    let [socket, setSocket] = useState(null);
+    let [isPriority, setIsPriority] = useState(false);
+
+    const togglePriority = () => {
+        if(isPriority)
+        setIsPriority(false);
+        else setIsPriority(true);
```

Comment:
this could be written as: `setIsPriority(prev => !prev);`

Reply:
This makes sense! Will change it.